client/src/utils/queries.js

```diff
@@ -1,11 +1,50 @@
 import { gql } from '@apollo/client';
 
-export const QUERY_PROFILE = gql`
-  query user($username: String!) {
-    user(username: $username) {
+export const QUERY_ME = gql`
+  query Me {
+    me {
       _id
       username
       email
+      petCount
+      pets {
+        _id
+        name
+      }
     }
   }
-`;
\ No newline at end of file
+`
+
+export const QUERY_PET = gql`
+  query pet($petId: String!) {
+    pet(petId: $petId) {
+      _id
+      name
+      species
+      breed
+      age
+      bio
+      pic
+      owner {
+        _id
+        username
+      }
+      events {
+        _id
+        title
+      }
+    }
+  }
+;`
```

Comment:
Made a mistake on line 38: semicolon should be on the right

Please fix before pull